packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -42,10 +43,11 @@ export default class DedicatedServerDashboard {
     this.URLS = URLS;
     this.COMMIT_IMPRESSION_TRACKING_DATA = COMMIT_IMPRESSION_TRACKING_DATA;
     this.RECOMMIT_IMPRESSION_TRACKING_DATA = RECOMMIT_IMPRESSION_TRACKING_DATA;
-
     this.servicesStateLinks = {
       weathermap: WEATHERMAP_URL[this.user.country] || WEATHERMAP_URL.OTHERS,
-      vms: this.constants.vmsUrl,
+      vms:
+        this.constants.vmsUrl[this.user.country] ||
```

Comment:
**question:** Shouldn't we check for Subsidiary instead of country?

Reply:
Done Anoop